### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Zelinskyi-Serhii**
[DEMO LINK](https://Zelinskyi-Serhii.github.io/react_article/)

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -1 +1,31 @@
...
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+      <p data-cy="text">
+        {text}
+      </p>
+    </>
+  );
+}
```
Comment (nazarovoleksiy) [inline src/components/Article.jsx:31]: `
  return (
    <>
      <h2 className="title" data-cy="title">
        {title}
      </h2>
      <p className="subtitle">
        <time dateTime={dateISO} data-cy="date">
          {dateLocaleString}
        </time>
      </p>
      <p data-cy="text">
        {text}
      </p>
    </>
  );
}
`

**Review by nazarovoleksiy — APPROVED** [review]
Good job, but its better to add empty lines for better readability.

File: src/components/Article.jsx [hunk 2]
```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title,
+  date,
+  text,
+}) {
+  const today = date;
```
Comment (nazarovoleksiy) [inline src/components/Article.jsx:8]: You no need 
const today = date;

You already have date :

const dateISO = date.toISOString().slice(0, 10);
  const dateLocaleString = date.toLocaleDateString('en-US', {
    year: 'numeric',
    month: 'long',
    day: 'numeric',
  });

File: src/App.jsx [hunk 3]
```diff
@@ -8,36 +9,16 @@ export const firstArticle = {
...
-        <p data-cy="text">
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
-          assumenda magni adipisci esse quidem ratione ipsa labore vero illo
-          optio.
-        </p>
+        <Article
+          title={firstArticle.title}
+          date={firstArticle.date}
+          text={firstArticle.text}
+        />
```
Comment (sergii-nosachenko) [inline src/App.jsx:21]: Checklist #2

File: src/components/Article.jsx (hunk 2 above)
Comment (sergii-nosachenko) [inline src/components/Article.jsx:8]: Useless variable

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

File: src/App.jsx [hunk 4]
```diff
@@ -8,36 +9,12 @@ export const firstArticle = {
...
-            {dateLocaleString}
-          </time>
-        </p>
-
-        <p data-cy="text">
-          Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime
-          assumenda magni adipisci esse quidem ratione ipsa labore vero illo
-          optio.
-        </p>
+        <Article {...firstArticle} />
```
Comment (sergii-nosachenko) [inline src/App.jsx:17]: This approach is safe only with Typescript.

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍
Commented

**Pull request closed without merging**